rules/integrations/cloud_defend/execution_anomalous_process_executable_launched_inside_a_container_bbr.toml

```diff
@@ -0,0 +1,63 @@
+[metadata]
+creation_date = "2023/04/25"
+integration = ["cloud_defend"]
+maturity = "production"
+min_stack_comments = "New Integration: Cloud Defend"
+min_stack_version = "8.8.0"
+updated_date = "2023/04/25"
+
+[rule]
+author = ["Elastic"]
+description = """Indicates a process executable launched inside a container and not previously seen in the last 15 days."""
+false_positives = [ ]
+from = "now-180s"
+index = [ "logs-cloud_defend*" ]
+interval = "2m"
+language = "kuery"
+license = "Elastic License v2"
+name = "Anomalous Process Executable Run Inside a Container"
+risk_score = 21
+rule_id = "c28bd942-ed9a-49fa-8ead-5c5f7aab97a0"
+severity = "low"
+tags = [
+  "Elastic",
+  "Host",
+  "Linux",
+  "Threat Detection",
+  "BBR",
+  "Execution",
+  "Container"
+]
+building_block_type = "default"
+timestamp_override = "event.ingested"
+type = "new_terms"
+
+query = '''
+process.entry_leader.entry_meta.type : "container" and 
+  event.type : "start" and event.action : "fork" and 
+  event.action : "exec" and (
+    (event.type: "end" and process.entry_leader.same_as_process:true) or 
+    (process.entry_leader.same_as_process:false)
+    ) and process.executable:*
```

Comment:
is it normal to have an and with two different event.action values ?  the following query is not equivalent ? 

```
process.entry_leader.entry_meta.type : "container" and 
    (process.entry_leader.same_as_process:true or process.entry_leader.same_as_process:false) and process.executable:*
```

Reply:
@Samirbous I'm reworking these queries now that I've received some clarity about the fields. However, there are multiple event.actions in most of the process events for the container events because of a burst filter that consolidates events that occur within ~1 sec of each other and contain all the same fields. So my thoughts here were to look for events with both "fork" and "exec" but to include the additional event.action: "end" if the process.entry_leader.same_as_process== true. But if the process.entry_leader.same_as_process== false then I'm just looking for fork and exec actions. I realize that this doesn't quite work because I would still capture "fork, exec, end" actions for process.entry_leader.same_as_process== false since I haven't actually excluded the "end" action. Again, I'm reworking these rules and they'll be less confusing.